### PR TITLE
Merge dev branch into main

### PR DESCRIPTION
**Pull request by Victoria129**
We have completed the requirements for this project and are now ready to merge into main.

### CODE REVIEW

File: src/component/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <Link to="/">Rockets</Link>
+          </li>
+          <li>
+            <Link to="/Missions">Missions</Link>
+          </li>
+          <li>
+            <span>|</span>
+          </li>
+          <li>
+            <Link to="/Profile">My Profile</Link>
```
Comment (prolajumokeoni) [inline src/component/Navbar.js:23]: - [ ] Based on the project requirements,  `<NavLink />` should be used for navigation links, NavLink helps  to highlight the active link by providing an attribute called `activeClassName={yourClassName}`

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @gsmalik030 and @Victoria129 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Design follows wireframe design
✔️  Data fetched from SpaceX API endpoints



## Required Changes ♻️

- [ ]  Kindly make sure that the tests written are passing and add more unit tests to your application, please  ensure  you write the new unit tests with React Testing Library and Jest

Screenshot_751

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly make sure that for future projects all pull requests created should be reviewed before merging, you currently have some pull requests that were not reviewed before merging, Code reviewing will help you become a better programmer, I suggest you do this for future projects.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/component/Navbar.js (hunk 1 above)
Comment (Victoria129) [reply src/component/Navbar.js]: Fixed

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**